Tests/Orso.Arpa.Api.Tests/IntegrationTests/AppointmentsControllerTests.cs

```diff
@@ -274,7 +274,7 @@ public async Task Should_Set_Dates()
             staffParticipation.Participation = null;
             expectedDto.Participations.Add(staffParticipation);
             AppointmentParticipationListItemDto adminParticipation = AppointmentDtoData.AdminParticipation;
-            adminParticipation.MusicianProfiles.Add(new MusicianProfileDto { IsProfessional = false, SectionName = "Soprano 2" });
+            adminParticipation.MusicianProfiles.Add(new MusicianProfileDto { SectionName = "Soprano 2" });
```

Comment:
please check your test, does my change fully heal the test or is something missing now?

Reply:
Tests have passed now